### PR TITLE
Nicolas - SF Automation - Master - LPD-36648

### PR DESCRIPTION
**Pull request by MicaelleSilvaa**
Ticket: https://liferay.atlassian.net/browse/LPD-36648

### CODE REVIEW

**Comment by MicaelleSilvaa** [comment]
@nicolas-sss

**Comment by MicaelleSilvaa** [comment]
https://github.com/liferay-devtools/liferay-portal/pull/237

**Comment by MicaelleSilvaa** [comment]
ci:test:hotel

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

**Comment by nicolas-sss** [comment]
ci:test:sf

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 1]
```diff
@@ -1307,6 +1309,9 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 
 	@Reference
 	private CPInstanceService _cpInstanceService;
+	
+	@Reference
+	private DDMFormViewFormInstanceRecordsDisplayContext _ddmFormViewFormInstanceRecordsDisplayContext;
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:1314]: @MicaelleSilvaa please create a method receiving this as a parameter. Being a parameter removes the "_" of the variable name.

**Review by nicolas-sss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**